PracticeTemplate/Controllers/Ex6/Ex6ViewController.swift

```diff
@@ -24,8 +66,5 @@ class Ex6ViewController: BaseViewController {
         self.title = "Ex6"
```

Comment:
Đang code ở chính nó nên không cần `self`
```suggestion
       title = "Ex6"
```